### PR TITLE
[tune] Try to enable PTL, SKlearn tests

### PR DESCRIPTION
**Pull request by richardliaw**
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: python/ray/util/sgd/torch/ptl_operator.py [hunk 1]
```diff
@@ -266,7 +269,8 @@ def train_epoch(self, iterator, info):
                 return_output = meter_collection.summary()
 
         if self.is_function_implemented("on_train_epoch_end", model):
-            model.on_train_epoch_end()
+            model.on_train_epoch_end(
```
Comment (richardliaw) [inline python/ray/util/sgd/torch/ptl_operator.py:272]: @amogkam can you please look at this and let me know if this is right?

**Comment by amogkam** [comment]
I will look over this by EOD

**Review by amogkam — APPROVED** [review]
Confirmed the PTL 1.0 changes and PR looks good to me!

**Comment by richardliaw** [comment]
great, thanks Amog!

**Pull request merged**